app/controllers/product_review_videos/streaming_urls_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class ProductReviewVideos::StreamingUrlsController < ApplicationController
+  before_action :set_product_review_video
+  after_action :verify_authorized
+
+  def index
+    return head :unauthorized unless authorized?
+
+    render json: {
+      streaming_urls: [
+        product_review_video_stream_path(
+          @product_review_video.external_id,
+          format: :smil
+        ),
+        @product_review_video.video_file.signed_download_url
+      ]
```

Comment:
This follows

https://github.com/antiwork/gumroad/blob/ff5ec881e28c0d59239dacb14f124271320a777a/app/models/url_redirect.rb#L216

I want to eventually consolidate these into `Streamable`. But need to take a closer look at the existing & future usage patterns.